gapic-generator/lib/gapic/schema/wrappers.rb

```diff
@@ -379,7 +371,7 @@ def ruby_package
       # @return [Google::Api::Package] Packaging information.
       #   See `google/api/client.proto`.
       def client_package
-        options[:".google.api.client_package"] if options
+        options[:".google.api.package"] if options
```

Comment:
Does this actually exist? It's no longer in `google/api/client.proto` as the line above states.

Reply:
This method was supposed to be removed but wasn't. Removed.